Esri/arcgis-maps-sdk-swift-samples
# [New] Add web tiled layer

## Description

This PR implements `Add web tiled layer` in `Layers` category.
URL to [README](https://devtopia.esri.com/runtime/common-samples/tree/main/designs/AddWebTiledLayer)

## Linked Issue(s)

- `swift/issues/`

## How To Test
Run the sample and a map will appear. As you navigate the map, map tiles will be fetched automatically and displayed on the map.

## Screenshots

![set-reference-scale](https://raw.githubusercontent.com/Esri/arcgis-maps-sdk-swift-samples/9b5e60c45c6a5e1fef9c36acc959e006b0042597/Shared/Samples/Add%20web%20tiled%20layer/add-web-tiled-layer.png)
